app/views/batch_imports/_form_metadata.html.erb

```diff
@@ -9,11 +9,11 @@
         input_html: { class: 'form-control' } %>
 
     <div class="with-headroom" style="padding-bottom:25px;">
-        <label>CSV Metadata Source</label><br />
```

Comment:
It sounds good and it's done in [commit](https://github.com/ucsdlib/horton/pull/82/commits/f108d921a783f48d57256ca0e4a9ca1fa666428a). Thanks.